.circleci/config.yml

```
@@ -2,10 +2,82 @@ version: 2.1
 orbs:
   architect: giantswarm/architect@0.2.0
 
+e2eTest: &e2eTest
+  machine: true
+  working_directory: /home/circleci/.go_workspace/src/github.com/giantswarm/metrics-server-app
+  steps:
+    - checkout
+
+    - attach_workspace:
+        at: .
+
+    - run:
+        name: Get e2ectl binary
+        command: |
+          wget -q $(curl -sS https://api.github.com/repos/giantswarm/e2ectl/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
+          chmod +x ./e2ectl
+    - run:
+        name: Init cluster
+        command: |
+          ./e2ectl cluster create --worker-count=3
```

```suggestion
          ./e2ectl cluster create
```

We use 3 workers in IC because it has node anti affinities. metrics-server is single pod so I think a single node cluster is fine.

Also not related to this but why doesn't the deployment set the replica count? 😕

---

You mean, the number of replica in deployment spec? I didn't know that it's not there yet. 
It's default to 1 from API spec. https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#deploymentspec-v1-apps

Should we add it though?

---

In KSM we set it. Up to you as it's not related to your change. I'd just like us to be consistent. 🙂 

https://github.com/giantswarm/kube-state-metrics-app/blob/f70aac02624394e49cc780e73ea4ca0e597f634c/helm/kube-state-metrics-app/values.yaml#L10

---

Sure I will add it